src/app/(auth)/login/page.tsx

```diff
@@ -23,20 +23,24 @@ export default function Login() {
     });
 
     if (error) {
-      throw new Error(`An error occurred trying to sign in: ${error.message}`);
+      setErrorMessage(error.message);
+    } else {
+      push('/');
     }
-    push('/');
   };
 
   return (
     <>
-      <H1>Log In</H1>
-      <SpacerDiv>
-        <QuestionsDiv>
+      <SpacerDiv $gap={0.625}>
+        <H1>Log In</H1>
+        {errorMessage !== '' && <P $color={COLORS.redMid}>{errorMessage}</P>}
```

Comment:
Can we make it so that the introduction of the error message does not shift around other components? (i.e. have it still take up the same amount of space when an error is not present?)

Reply:
Ahh @davidqing6432 what do you think? I think the current implementation is consistent with figma.

Reply:
oop i didn't even cross-reference with the figma, that's my bad! in that case we can leave this be :)